web/app/view/ReportController.js

```diff
@@ -60,6 +60,15 @@ Ext.define('Traccar.view.ReportController', {
         dialog.show();
     },
 
+    updateButtons: function () {
+        var reportType, disabled, devices;
+        reportType = this.lookupReference('reportTypeField').getValue();
+        devices = this.deviceId && this.deviceId.length !== 0 || this.groupId && this.groupId.length !== 0;
+        disabled = !reportType || !devices;
```

Comment:
It configured by default from 30 minutes before to now. And can't be not configured.